blocks/subscriptions-block/components/SubscriptionDialog/styles.scss

```diff
@@ -0,0 +1,83 @@
+.xpmedia-subscription-dialog {
+  display: grid;
+  grid-template-columns: map-get($spacers, 'md') repeat(10, 1fr) map-get($spacers, 'md');
+  grid-template-rows: auto;
+  // this could be defined here or at the parent level
+  // margin: map-get($spacers, 'xxl') auto;
+  text-align: center;
+  @media screen and (min-width: map-get($grid-breakpoints, 'md')) {
+    grid-template-columns: repeat(12, 1fr);
+    // this could be defined here or at the parent level
+    // margin: map-get($spacers, 'xl') auto;
```

Comment:
comment meant to be included?

Reply:
I had it commented out because Matt had concerns that it was for the parent, but in thinking about it, I think it's appropriate here so uncommenting.

Reply:
Uncommenting this will make us have to revisit this if and when we want to place this component inside a different paywall container. Presentational components should aim to have no layout spacing within them